src/tile/tile.scss

```diff
@@ -313,3 +320,16 @@
     @include visually-hidden;
   }
 }
+
+@mixin iui-responsive-grid {
+  --_iui-grid-item-min-width: #{$tile-min-width};
```

Comment:
Why did you make this a private variable instead of just using `$tile-min-width`?